presto-main/src/main/java/com/facebook/presto/operator/HashAggregationOperator.java

```diff
@@ -299,6 +312,10 @@ public OperatorFactory duplicate()
     // for yield when memory is not available
     private Work<?> unfinishedWork;
 
+    private long inputBytesProcessed;
+    private long inputRowsProcessed;
+    private long uniqueRowsProduced;
```

Comment:
nit: Will hash agg ever produce non unique rows ? ie, Is the word unique redundant ?